modules.d/06rngd/module-setup.sh

```diff
@@ -32,7 +32,12 @@ depends() {
 
 install() {
     inst rngd
-    inst_simple "${moddir}/rngd.service" "${systemdsystemunitdir}/rngd.service"
+    inst_simple "${systemdsystemunitdir}/rngd.service"
+
+    if [ -r /etc/sysconfig ]; then
```

Comment:
The check here does not make much sense. It would be checking if the dir is readable in the normal system, not in the generated initrd. I think it can be safely dropped.

Reply:
Sorry, my mistake :) I forgot to add `/rngd`.

Reply:
Fix incomming.